src/webpack/index.js

```diff
@@ -74,9 +159,46 @@ class URLImportPlugin {
           name = path;
         }
 
+        // console.log('stats', stats);
+
+        if (externalModules[chunk.id]) {
```

Comment:
@theKashey pretty much this loop. Itll tell you about the dependencies on each module. 
`const module of dependencyModuleSet.chunksIterable` lets you loop over the modules in the dependency set. I use (or will be) to get `module.chunks.foreach((chunk)=>chunk.files)`

Reply:
There's probably a lower level hook you would tap, but inside those chunks are the orderings they need to be loaded in. If a dependency matrix was created, you be able to look up any order